flight/intake_gps.py

```diff
@@ -0,0 +1,79 @@
+"""
+Contains the extract_gps() function for extracting data out of
+a provided target waypoint data JSON file for the Argonia Cup competition.
+"""
+from typing import Any, NamedTuple, Tuple
+import json
+import argparse
```

Comment:
Should import argparse at the top of your main. Don't need it for the whole file